assets/js/components/notifications/CTA.js

```diff
@@ -85,6 +86,7 @@ CTA.defaultProps = {
 	ctaLabel: '',
 	error: false,
 	onClick: () => {},
+	ctaLinkExternal: false,
```

Comment:
No need to set `false` by default for this property. It should be `undefined` if not set.